extensionsGallery-insider.json

```diff
@@ -5036,30 +5036,30 @@
 					},
 					"versions": [
 						{
-							"version": "1.82.15",
```

Comment:
@lewis-sanchez You should be updating the extensionsGallery file for the rc1 branch, not insiders

Reply:
Oops. I'll fix it.